packages/check/package.js

```diff
@@ -4,15 +4,17 @@ Package.describe({
 });
 
 Package.onUse(function (api) {
-  api.use(['underscore', 'ejson'], ['client', 'server']);
-  api.use('jquery', 'client');
+  api.use(['underscore', 'modules', 'ejson'], ['client', 'server']);
```

Comment:
I tried that, but got an error with a circular reference that ecmascript package depended on check. I'll just change `const` to `var`.